### DISCUSSION TITLE
自分の記事をタグで検索したい

### DISCUSSION THREAD

**Discussion by duri0214** [increments/qiita-discussions#144] [要望・アイデア・不具合]
自分の試行錯誤をよくQiitaに書くと、だんだん増えてきてページングが深くなってしまう。あれ？たしか書いたな？みたいなことがある。
無理やり「ピックアップ記事」にしたりして対応しているが、
タグでインクリメント検索できると嬉しい

**Comment by kyntk** [comment]
@duri0214 
ご意見ありがとうございます。

現状だとマイページでの検索はできないのですが、検索ページがありますのでこちらでは検索することができます 🙇🏽 

`user:kyntk` のようにすると自分の投稿した記事に絞って検索することができますので、こちらをご利用ください。
https://qiita.com/search?sort=&q=user%3Akyntk+ruby

**Reply by duri0214** [reply]
この検索で十分なので、こちらクローズで大丈夫です。

**Reply by yoshi389111** [reply]
ちょっとした思いつきですが、例えばユーザーの記事一覧の右上あたりに検索（虫眼鏡）アイコンを用意して、それをクリックすると `user:ユーザー名` の検索画面に遷移できると、とっても便利な気がします。

![image](https://user-images.githubusercontent.com/15214393/155283623-d8ccd9d8-bf96-46d8-a9f7-6be80f54e41b.png)

↓（虫眼鏡アイコンをクリックしたら）

![image](https://user-images.githubusercontent.com/15214393/155283044-7323087c-7915-4769-a991-fb9ced9504b5.png)

（必要ならタグとかキーワードとかを手動で追加して再検索）

**Reply by duri0214** [reply]
そこまでやってくれるとたしかに嬉しいですね。
`user:me`  とかでも嬉しいですね（私の名前 `yoshitakaokada` でいえば `yoshi` も `yoshitaka` も他人がいるので

**Reply by gilly135** [reply]
@duri0214  @yoshi389111 
ご意見ありがとうございました。
時間が空いてしまいましたが、ユーザーページから、閲覧中のユーザーを検索条件に指定した検索結果ページに遷移できるようにしました！

詳細はリリースノートをご確認ください。
https://qiita.com/release-notes/2024-07-24

ご利用いただき、またご要望などございましたらお知らせいただけると嬉しいです 🙏
